test/expressions.tests.ts

```diff
@@ -4591,7 +4591,7 @@ select x.Key1;`);
           Token.Identifiers.LocalName("b"),
           Token.Keywords.Control.When,
           Token.Punctuation.OpenParen,
-          Token.Variables.ReadWrite("double"),
```

Comment:
The old test code does not make sense. Why is this `double` keyword the only instance in this entire test code marked as `Variables.ReadWrite`?
All others are marked correctly as `PrimitiveType.Double`.

Reply:
Looks like it was just documenting the behavior, not the desired behavior.